docs/sources/tutorials/get-started.md

```diff
@@ -71,17 +87,169 @@ services:
           editable: false
         EOF
         /run.sh
-    image: grafana/grafana:latest
+    image: grafana/grafana:11.0.0
     ports:
       - "3000:3000"
 ```
 
-After running `docker-compose up`, open [http://localhost:3000](http://localhost:3000) in your browser to view the Grafana UI.
+Run `docker compose up` to start your Docker container and open [http://localhost:3000](http://localhost:3000) in your browser to view the Grafana UI.
+
+ {{< admonition type="note" >}}
+If you the following error when you start your Docker container, `docker: 'compose' is not a docker command`, use the command `docker-compose up` to start your Docker container.
+ {{< /admonition >}}
+
+## Configure {{< param "PRODUCT_NAME" >}}
```

Comment:
```suggestion
## Configure {{% param "PRODUCT_NAME" %}}
```

Missed this in earlier reviews. We need to use `%` in the shortcode syntax when they are used in headings.